pkg/buildpack/buildpack_tar_writer_test.go

```diff
@@ -44,8 +44,9 @@ func testBuildModuleWriter(t *testing.T, when spec.G, it spec.S) {
 	)
 
 	it.Before(func() {
-		logger = logging.NewLogWithWriters(&outBuf, &outBuf, logging.WithVerbose())
+		h.SkipIf(t, runtime.GOOS != "linux", "Skipped on non-linux")
```

Comment:
What happens if we try to use this feature on Windows?

Reply:
Uggh disregard, I thought I was reviewing the PR but it was only one commit 😞

Reply:
I have to check all the Windows support, it's been hard to test the feature only in the ci runners, that's why I am skipping the tests there. But, I am trying to set up a VM to give it a try